modules/ROOT/pages/oauth-dance-about.adoc

```diff
@@ -3,37 +3,42 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The authentication process performed by the Mule OAuth 2.0 provider, API, and client application conforms to RFC 6749. This process is called the OAuth Dance.
+The OAuth Dance is the authentication process performed by the Mule OAuth 2.0 provider, API, and client application. This authentication process conforms to RFC 6749.
 
-image::oauth-policy1.png[oauth+policy1,height=366,width=640]
+== OAuth Dance Authentication Flow
 
-. A token is requested to the provider, specifying client application credentials as needed (the credentials correspond to a Client ID and secret generated by API Platform).
-. Mule OAuth 2.0 provider returns a token.
-. An HTTP request is performed against the protected resource (ie resource where the OAuth 2.0 Access Token Enforcement policy is applied), including the token either as an authentication header or a query parameter.
-- The protected resource is linked with the API definition through Autodiscovery.
-. The OAuth 2.0 Access Token Enforcement policy intercepts this request and communicates with the Provider to validate the token.
+The OAuth Dance authentication flow is illustrated in the following screenshot:
+image::mule-oauth-provider-dance.png[oauth+policy1,height=366,width=640]
+
+. A token request is sent to the provider, specifying client application credentials as needed. The credentials correspond to a client ID and secret generated by API Platform.
+. The Mule OAuth 2.0 provider returns a token.
+. An HTTP request is performed against the protected resource to which the OAuth 2.0 Access Token Enforcement policy is applied.
++
+The protected resource *must* be linked with the API definition through Autodiscovery.
```

Comment:
Do not use bold for emphasis. Let the context provide that ("must" is a pretty strong term alone), or under rare circumstances, use italic (_ _).

Reply:
Done! Thanks!